app/editor/src/features/admin/ingests/configurations/Newspaper.tsx

```diff
@@ -210,7 +211,7 @@ export const Newspaper: React.FC = (props) => {
         label="Default Source"
         name="configuration.defaultSource"
         tooltip="Source to use for publications missing from Sources list. If not set, articles will be discarded"
-        options={sources}
+        options={filterEnabled(sources)}
```

Comment:
If the Content has a value that is no longer enabled this will not work.

Reply:
The linting issues should have been resolved now, and the code above are updated, thanks